src/NHibernate/Persister/Entity/IEntityPersister.cs

```diff
@@ -404,6 +404,11 @@ public partial interface IEntityPersister : IOptimisticCacheSource
 		/// </summary>
 		bool HasCache { get; }
 
+		/// <summary>
+		/// Does this class tracked with UpdateTimestampsCache? Configured via entity mapping Cache.Never()
+		/// </summary>
+		bool HasUpdateTimestampsCache { get; }
```

Comment:
This is also a breaking change, you can avoid the breaking change by adding an extension method instead. Maybe  `SupportsQueryCache` would be a better name for this property.

Reply:
SupportsQueryCache  ok